JesusOmarR/react-apprenticeship-capstone-2
# React Apprenticeship Capstone 2

## Description
```
React apprenticeship capstone 2
Created the App to fetch the picture of the day from the NASA API
Created a date picker to select a date and fetch the image of that day
Also I added styles to the app to have a better UI, and its responsive
Created a custom hook to fetch the data from the API

- 
```
## Type of Change

```
Delete options that are not relevant.
```


## How should this be tested?
```
Run npm run tests to see all the test cases and verify if they have a good implementation
```
## What is the deliverable?
```
React apprenticeship 2
```
## Screenshots and links
```
<img width="1727" alt="image" src="https://user-images.githubusercontent.com/97203630/150906663-6862277f-7156-4568-b76f-8a45a5cdf95e.png">
```
## Notes
```
Please provide any additional information or any concern you may want to be reviewed more in detail
```
## Checklist
- [ x] I added the necessary documentation, if appropriate.
- [ x] I added tests to prove that my fix is effective or my deliverable works.
- [ x] I reviewed existing Pull Requests before submitting mine.
- [ x] I have commented my code, particularly in hard-to-understand areas
